main.py

```diff
@@ -38,7 +39,6 @@
 
 # setting up path hierarchy for commands to load
```

Comment:
why not remove this while you are at it since your changes deprecate this?

Reply:
@TitanVJ

Reply:
done